lib/saml/health_status.rb

```diff
@@ -46,7 +46,7 @@ def idp_certs_valid?
 
         begin
           signing_cert    = SettingsService.merged_saml_settings&.idp_cert_multi&.dig(:signing)&.first
-          encryption_cert = SettingsService.merged_saml_settings&.idp_cert_multi&.dig(:encryption)&.first
+          encryption_cert = SettingsService.merged_saml_settings&.idp_cert_multi.dig(:encryption).first
```

Comment:
Nice change while you're here. Also seems like a separate fixup PR/commit to me.

Reply:
Hey there, the purpose of this rubocop change was just for testing. The goal was to break the jenkins build to verify that stdout flush was working properly 😸

Reply:
Gotcha. But... don't `&.` and `dig` accomplish similar things? And thus seems weird the way this is written to me at least.

In any case, glad its not part of the PR 😄